plugin.cc

```diff
@@ -103,13 +104,14 @@ MLDB::Plugin * mldbPluginEnter(MldbServer * server)
 namespace {
 
 /// Register our function with MLDB.  We say here that our function
-/// hello.world takes an integer for configuration (which is ignored),
+/// helloWorld takes an integer for configuration (which is ignored),
 /// has the given description and has no documentation available.
 
-static RegisterFunctionType<HelloWorldFunction, int>
-regHelloWorldFunction("hello.world",
-                    "Sample function that always returns hello = \"world\"",
-                    "No documentation available");
-
+Package package("mldb_sample_plugin");
 
+static RegisterFunctionType<HelloWorldFunction, int>
+regHelloWorldFunction(package,
+                      "helloWorld",
```

Comment:
I don't understant. With the current form I expect something like
```
SELECT helloWorld(1)
```
With the form hello.world, would the call be
```
SELECT hello.world(1)
```
?